components/fullenrich/actions/enrich-contact/enrich-contact.mjs

```diff
@@ -0,0 +1,108 @@
+import { ConfigurationError } from "@pipedream/platform";
+import app from "../../fullenrich.app.mjs";
+
+export default {
+  key: "fullenrich-enrich-contact",
+  name: "Enrich Contact",
+  description: "Starts the enrichment process for a specified contact. [See the documentation](https://docs.fullenrich.com/startbulk)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    name: {
+      type: "string",
+      label: "Name",
+      description: "The name of the action.",
+    },
+    webhookUrl: {
+      type: "string",
+      label: "Webhook URL",
+      description: "The Webhook URL that will be triggered when the enrichment is done. Optional.",
+      optional: true,
+    },
```

Comment:
(Optional) Could use an optional `callbackWithRerun` prop and Pipedream's `$.flow.rerun()` to pause the workflow until the result is ready. See [twin-browse](https://github.com/PipedreamHQ/pipedream/blob/master/components/twin/actions/browse/browse.mjs) for an example.